HW_25.10/HW_25.10/appsettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "DataFile": "This is first task",
+  "DataFile1" :  "D:\\TeachMeSkills\\TeachMeSkillls\\TeachMeSkills\\HW_25.10\\Primer.txt"
```

Comment:
Always use relative pathes
This path will work only on your computer, never use full paths
"Primer.txt" would be enough and you would need to find the way to reach the file in your code